src/compas_fab/robots/robot.py

```diff
@@ -31,21 +31,24 @@ class Robot(Data):
     upon the model described in the class :class:`compas_robots.RobotModel` of
     the **COMPAS** framework.
 
-    Attributes
+    Parameters
     ----------
-    model : :class:`RobotModel`
+    model : :class:`compas_robots.RobotModel`
         The robot model, usually created from an URDF structure.
     scene_object : :class:`compas_robots.scene.BaseRobotModelObject`
         Instance of the scene object used to visualize the robot model. Defaults to ``None``.
-    semantics : :class:`compas_fab.robots.RobotSemantics`
+    semantics : :class:`~compas_fab.robots.RobotSemantics`
         The semantic model of the robot. Defaults to ``None``.
-    client : :class:`compas_fab.backends.interfaces.ClientInterface`
+    client : :class:`~compas_fab.backends.interfaces.ClientInterface`
         The backend client to use for communication,
-        e.g. :class:`compas_fab.backends.RosClient`
+        e.g. :class:`~compas_fab.backends.RosClient`
+
+    Attributes
+    ----------
```

Comment:
I agree with documenting `Parameters`, but now this part of `Attribuets` is incomplete now, because it only lists attributes not present in the constructor's part.

Reply:
So I guess some of the attributes will be a repeat of the parameters. Yea?

Reply:
Fixed.